lib/models/device.coffee

```diff
@@ -509,6 +544,76 @@ exports.move = (uuid, application, callback) ->
 	.nodeify(callback)
 
 ###*
+# @summary Start application on device
+# @name start
+# @public
+# @function
+# @memberof resin.models.device
+#
+# @param {String} uuid - device uuid
+# @fulfil {String} - application container id
+# @returns {Promise}
+#
+# @example
+# resin.models.device.start('7cf02a6').then(function(containerId) {
+# 	console.log(containerId);
+# });
+#
+# @example
+# resin.models.device.start('7cf02a6', function(error, containerId) {
+# 	if (error) throw error;
+# 	console.log(containerId);
+# });
+###
+exports.start = (uuid, callback) ->
```

Comment:
Right, so there is already a `.restart()` method, which has the semantics of restarting the application, so I kept `.start()` and `.stop()` for consistency. Also, at this point we assume single-container apps, since the supervisor APIs do not support taking a containerId as an argument. I am fine with renaming the function if you feel strongly about it though. 

We could also have `.restartApplication()`, `.stopApplication()`, `.startApplication()`